services/analyze-sequelize-tables.js

```diff
@@ -10,8 +10,8 @@ let tableForeignKeysAnalyzer;
 let columnTypeGetter;
 
 function isUnderscored(fields) {
```

Comment:
This comment is for `analyze-sequelize-tables` dependencies.
This is to keep `services` folder simple and easy to read.

Please rename `column-type-getter` to `analyze-sequelize-column-type-getter`
Please rename `table-foreign-keys-analyzer` to `analyze-sequelize-foreign-key-getter`

Please create an `analyzer` folder, and put all `analyze-sequelize-...` files inside it.

Reply:
(Note that after you move every files into `analyzer`, the analyze prefix is useless.)